src/Stratis.FederatedPeg.Features.FederationGateway/TargetChain/CrossChainTransferStore.cs

```diff
@@ -1066,7 +1069,9 @@ private void PutTransfer(DBreeze.Transactions.Transaction dbreezeTransaction, IC
         {
             Guard.NotNull(crossChainTransfer, nameof(crossChainTransfer));
 
-            dbreezeTransaction.Insert<byte[], ICrossChainTransfer>(transferTableName, crossChainTransfer.DepositTransactionId.ToBytes(), crossChainTransfer);
+            byte[] crossChainTransferBytes = this.dBreezeSerializer.Serialize(crossChainTransfer);
```

Comment:
this.dbreezeSerializer is not required if doing it this way:

byte[] crossChainTransferBytes = crossChainTransfer.ToBytes(this.network.Consensus.ConsensusFactory);